### DISCUSSION TITLE
[PSD8] Exercise 2.31/2.32

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#162] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210060047-b72f3564-f6ea-4d01-9638-9f3c60ddfea7.png)

![image](https://user-images.githubusercontent.com/69389461/210060064-672e5433-d0ea-441e-b748-38d46fcdb528.png)

**Comment by Elia-Belli** [comment]
$Y = A \bar B \bar C \bar D + CD + BD + \bar A D$
![K-map  PSD8](https://user-images.githubusercontent.com/107944476/211599357-aeb4e215-304c-4e11-84a4-daeaa2b06491.png) 
![CC](https://user-images.githubusercontent.com/107944476/212953678-fd8150cd-caa8-4086-9313-ceac9c37f8c9.jpg)

**Comment by fraxb04** [comment]
corretto
